orcestra_book/plans/HALO-20240831a.md

```diff
@@ -315,6 +315,10 @@ if os.path.isdir(localdir):
             file.write (f'\n{position:22s} {person}')
 ```
 
-```{code-cell} ipython3
 
+```{code-cell} python3
```

Comment:
I'd keep this with `ipython3` as it doesn't really matter, but when using the markdown files in interactive notebooks, they will change it automatically to `ipython3`.
````suggestion
```{code-cell} ipython3
````